vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -317,7 +222,7 @@ void Contribution::StartMonthlyContribution() {
                 _1,
                 _2));
 
-  SetTimer(&delay_ac_timer_id, 10);
+  SetTimer(&delay_ac_timer_id, 30);
```

Comment:
Change this to reduce potential race condition between recurring tips and AC. Will be fully fixed with https://github.com/brave/brave-browser/issues/6288